wallet/i18n/es.json

```diff
@@ -351,5 +351,9 @@
   "free": "Tasuta",
   "claim_free_nft": "Nõudke oma tasuta NFT-d",
   "before_too_late": "Enne kui on liiga hilja!",
-  "please_try_again_later": "Por favor, inténtelo de nuevo más tarde"
+  "please_try_again_later": "Por favor, inténtelo de nuevo más tarde",
+  "buying_pylon_points": "Puntos Pylons comprados",
```

Comment:
Should comprados be capitalized?

Reply:
@heyitsjk

Reply:
@kjawadDeveloper2  "Puntos Pylons Comprados" works here too, yes

Reply:
@Amean resolved.